blaseball_mike/chronicler.py

```diff
@@ -133,3 +133,23 @@ def get_tribute_updates(before=None, after=None, order=None, count=None):
         params["count"] = count
 
     return paged_get(f'{BASE_URL}/tributes/hourly', params=params, session=cached_session)
+
+def time_map(season=0, tournament=-1, day=0, include_nongame=False):
+    # Return start & end timestamps for a given gameday
```

Comment:
can you add a docstring for usage then generate the docs?

Reply:
done. This also caught the Coffee 3s change for the docs as well